federated_aws_cli/login.py

```diff
@@ -261,24 +281,31 @@ def callback(self, code, state, token=None, **kwargs):
                     print(get_aws_shared_credentials(path))
```

Comment:
It looks like this loses the [changes](https://github.com/mozilla-iam/federated-aws-cli/blob/aa111b13b5781990e75d96f77cc0fe409bc4b96a/federated_aws_cli/login.py#L255-L266) that came from #97

Reply:
Yes, I thought you were going to handle the merge conflict?

Reply:
No, my PR was merged first, so your branch needs to merge master back in to conflict resolve.